applications/crossbar/src/modules/provisioner_v5.erl

```diff
@@ -544,9 +544,15 @@ req_headers(Token) ->
          ,{"User-Agent", wh_util:to_list(erlang:node())}
         ]).
 
--spec get_cluster_id() -> 'undefined' | string().
+-spec get_cluster_id() -> string().
 get_cluster_id() ->
-    whapps_config:get_string(?MOD_CONFIG_CAT, <<"cluster_id">>).
+    case whapps_config:get_string(?MOD_CONFIG_CAT, <<"cluster_id">>) of
```

Comment:
What for? `?DEFAULT_CLUSTER_ID` would only be used here,
and if it wasn't the macro would have a different value on every call…